packages/rstream/src/api.ts

```diff
@@ -152,3 +155,5 @@ export type StreamSource<T> = (sub: Stream<T>) => StreamCancel | void;
 export let LOGGER = NULL_LOGGER;
 
 export const setLogger = (logger: ILogger) => (LOGGER = logger);
+
+export const DUMMY: ISubscriber<any> = { next() {} };
```

Comment:
Is this supposed to be part of the public API?  Looks like it's just for internal use.

Reply:
hehe... nope, don't worry - this was just a temp workaround, it's already gone again... this whole PR is WIP